libraries/monitor/src/builtins.js

```diff
@@ -39,14 +40,20 @@ module.exports = [
                   that, it is not necessarily true that the endpoint was
                   _authorized_. You can tell that by the statusCode.`,
     fields: {
-      name: 'The name of the endpoint.',
-      resource: 'The path of the http request. Should match the one in auth audit log.',
-      method: 'The http method of the request. Should match the one in auth audit log.',
+      apiMethod: 'The name of the API method (in the format serviceName.apiVersion.name).',
+      resource: 'The path of the http request.',
+      method: 'The http method of the request.',
       statusCode: 'The http status code that the endpoint resolved with.',
-      duration: 'The duration in ms of the endpoint.',
-      public: 'Will be true if the endpoint requires no scopes.',
+      duration: 'The duration in ms of the call.',
+      public: 'True if the endpoint requires no scopes.',
       hasAuthed: 'Will be true if the request had req.authorize() called on it.',
-      clientId: 'The clientId that made the request if there was one.',
+      clientId: 'The clientId that made the request, if there was one.',
+      satisfyingScopes: `
+        The set of scopes posessed by the caller that were used to authorize this request, or [] if scopes were not required.
+
+        This set of scopes is intuitively "the minimal set of scopes required to satisfy the scope expression" but is not quite minimal in one sense:
```

Comment:
"minimal set of scopes possessed by the client"?

Reply:
It needs to have some reference to the API method.  How about "the minimal set of scopes required to satisfy the API method's authorization requirements"?

Reply:
The only problem I have with that is that it makes it seem like something we can know about a given endpoint without knowing the scopeset that we tried to evaluate it with.

Reply:
Good point -- I'll add some sentences.